pkg/cli/debug.go

```diff
@@ -230,8 +231,43 @@ func runDebugKeys(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	filter := func(kv storage.MVCCKeyValue) bool {
+		return true
+	}
+	switch debugCtx.filterKeys {
+	case showValuesFilter:
+		filter = func(kv storage.MVCCKeyValue) bool {
+			return kv.Key.IsValue()
+		}
+	case showIntentsFilter:
+		filter = func(kv storage.MVCCKeyValue) bool {
+			if kv.Key.IsValue() {
+				return false
+			}
+			var meta enginepb.MVCCMetadata
+			if err = protoutil.Unmarshal(kv.Value, &meta); err != nil {
+				return false
+			}
+			return meta.Txn != nil
```

Comment:
I think we should check that there's no timestamp as well (intents do not have timestamps as they're not written yet).